src/tests/ftest/util/dmg_utils.py

```diff
@@ -627,10 +661,12 @@ def system_start(self):
         """
         return self._get_result(("system", "start"))
 
-    def system_stop(self, force=False):
+    def system_stop(self, ranks=None, force=False):
```

Comment:
When adding new method arguments it is best to place them after existing arguments as any existing calls to the method which passed arguments positionally will break (https://jira.hpdd.intel.com/browse/DAOS-5638), e.g.
```
                # Stop all servers
                self.get_dmg_command().system_stop(True) 
```

Reply:
Done. Thanks for letting me know.